RLBotPack/VirxEB/gui.py

```diff
@@ -0,0 +1,133 @@
+from os import path
+from threading import Thread
+from tkinter import HORIZONTAL, BooleanVar, PhotoImage, StringVar, Tk, ttk
```

Comment:
Tkinter will fail to import for anybody using the old RLBotGUI installer, because it uses an embedded python which is lacking it. That represents a large percentage of our user base still.

To mitigate, please specify a requirements.txt file like I did here: https://github.com/RLBot/RLBotPack/commit/1ecfb5802025c8653a34753f80a4131a1f9ca328

That will cause a warning icon to appear in RLBotGUI for people who have the old launcher, and they will be given instructions on how to upgrade themselves if they want.

Reply:
Alr! I've done it 👍